From: Andrew Morton <akpm@linux-foundation.org>
Subject: maple_tree-add-mas_next_range-and-mas_find_range-interfaces-checkpatch-fixes
Date: Fri May 12 03:59:53 PM PDT 2023

ERROR: Macros with complex values should be enclosed in parentheses
#58: FILE: include/linux/maple_tree.h:544:
+#define mas_contiguous(__mas, __entry, __max) \
+	while (((__entry) = mas_find_range((__mas), (__max))) != NULL)

WARNING: suspect code indent for conditional statements (8, 12)
#285: FILE: lib/maple_tree.c:6136:
+	if (mas_find_setup(mas, max, &entry))
+	    return entry;

WARNING: Statements should start on a tabstop
#286: FILE: lib/maple_tree.c:6137:
+	    return entry;

total: 1 errors, 2 warnings, 284 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/maple_tree-add-mas_next_range-and-mas_find_range-interfaces.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: "Liam R. Howlett" <Liam.Howlett@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/maple_tree.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/maple_tree.c~maple_tree-add-mas_next_range-and-mas_find_range-interfaces-checkpatch-fixes
+++ a/lib/maple_tree.c
@@ -6134,7 +6134,7 @@ void *mas_find(struct ma_state *mas, uns
 	void *entry = NULL;
 
 	if (mas_find_setup(mas, max, &entry))
-	    return entry;
+		return entry;
 
 	/* Retries on dead nodes handled by mas_next_slot */
 	return mas_next_slot(mas, max, false);
_
